src/main/java/net/finmath/marketdata/products/Deposit.java

```diff
@@ -52,17 +52,26 @@ public double getValue(double evaluationTime, AnalyticModelInterface model) {
 			throw new IllegalArgumentException("No curve of the name " + discountCurveName + " and type DiscountCurveInterface was found in the model.");
 		}
 
-		double fixingDate	= schedule.getFixing(0);

```

Comment:
This change is inconsistent with a) the documentation and b) the method getRate(). Also I believe it would make more sense to use fixing and not start, since start and end are used for the calculation or periodLength only.